### ISSUE TITLE
NFS mounted local folder stopped syncing

### ISSUE THREAD

**Issue by jaywink** [owncloud/client#5529]
I have several local folders being synced and one folder that is mounted over NFS. This has worked fine for a few months. Sometime in the period of a few weeks ago, syncing has stopped as the ownCloud client has not started any more. I only realized it recently while wondering why a local file hadn't updated from another computer and then realized there is no ownCloud client icon in the task bar.

I've also tried the Nextcloud client from the PPA. Both versions are the same.

Server side is fine - I have other computers syncing this folder, though they have it locally. Only this machine has problems where (due to the size of the content) the folder is mounted over NFS.

### Expected behaviour
ownCloud or Nextcloud client should start and sync files.

### Actual behaviour
ownCloud and Nextcloud clients fail to start if there is an NFS mount as a sync folder. Removing said folder from `~/.local/share/data/Nextcloud/nextcloud.cfg` enables the client to start and syncing of the other folders work.

Also worth noting, when I decided to try Nextcloud client instead of ownCloud client, I had to re-add folders one by one. When I added the NFS mounted folder, the client hang and didn't sync or show in task bar after restart.

### Steps to reproduce
1. Add an NFS mounted folder as a sync folder.

### Server configuration
Operating system:
Ubuntu 16.04.2 LTS

Web server: 
Apache  2.4.7

Database:
PostgreSQL 9.3

PHP version:
5.5.9

ownCloud version:
Nextcloud 10.0.1

Storage backend (external storage):
-

### Client configuration
Client version:
ownCloud 2.2.4
Nextcloud 2.2.4
Nextcloud says "Built from Git revision eaeed0 on Feb 7 2017, 19:11:01 using Qt 4.8.7, OpenSSL 1.0.2g 1 Mar 2016"

Operating system:
Ubuntu 16.04.2

OS language:
?

Qt version used by client package (Linux only, see also Settings dialog):
4.8.7

Client package (From ownCloud or distro) (Linux only):
ownCloud from official 16.04 repo https://software.opensuse.org/download/package?project=isv:ownCloud:desktop&package=owncloud-client
Nextcloud from PPA https://launchpad.net/~nextcloud-devs/+archive/ubuntu/client

Installation path of client:
/usr/bin/owncloud
/usr/bin/nextcloud

### Logs

Client log file for this particular folder

    02-12 13:58:36:276 21328 OCC::FolderMan::addFolderInternal: Adding folder to Folder Map  OCC::Folder(0x1e3f5f0) "Music"
    02-12 13:58:36:276 21328 OCC::FolderWatcherPrivate::slotAddFolderRecursive: (+) Watcher: "/lmedia/Music/"
    02-12 13:58:37:623 21328 OCC::FolderWatcher::pathIsIgnored: * Ignoring file "/lmedia/Music/MP3/Mew/.fr-UJw59e"
    02-12 13:58:37:623 21328 OCC::FolderWatcherPrivate::slotAddFolderRecursive: * Not adding "/lmedia/Music/MP3/Mew/.fr-UJw59e"
    02-12 13:58:37:623 21328 OCC::FolderWatcher::pathIsIgnored: * Ignoring file "/lmedia/Music/MP3/Mew/.fr-xQHP93"
    02-12 13:58:37:624 21328 OCC::FolderWatcherPrivate::slotAddFolderRecursive: * Not adding "/lmedia/Music/MP3/Mew/.fr-xQHP93"
    02-12 13:58:37:632 21328 OCC::FolderWatcherPrivate::slotAddFolderRecursive:     `-> and 1383 subdirectories
    02-12 13:58:37:632 21328 OCC::FolderMan::slotScheduleSync: Schedule folder  "Music"  to sync!
    02-12 13:58:37:632 21328 OCC::FolderMan::slotScheduleSync: Folder is not ready to sync, not scheduled!
    02-12 13:58:37:632 21328 OCC::Folder::checkLocalPath: Checked local path ok
    02-12 13:58:37:632 21328 OCC::Folder::setIgnoredFiles: ==== adding system ignore list to csync: "/etc/ownCloud/sync-exclude.lst"
    02-12 13:58:37:632 21328 OCC::Folder::setIgnoredFiles: ==== adding user defined ignore list to csync: "/home/jaywink/.local/share/data/ownCloud/sync-exclude.lst"

Log ends with 

    02-12 13:58:37:858 21328 OCC::SyncJournalDb::checkConnect: sqlite3 version "3.11.0"
    02-12 13:58:37:858 21328 OCC::SyncJournalDb::checkConnect: sqlite3 journal_mode= "wal"
    02-12 13:58:37:858 21328 OCC::SyncJournalDb::commitInternal: void OCC::SyncJournalDb::commitInternal(const QString&, bool) Transaction commit  "checkConnect" and starting new transaction
    02-12 13:58:37:858 21328 OCC::SyncJournalDb::tableColumns: Columns in the current journal:  ("phash", "pathlen", "path", "inode", "uid", "gid", "mode", "modtime", "type", "md5", "fileid", "remotePerm", "filesize", "ignoredChildrenRemote", "contentChecksum", "contentChecksumTypeId")
    02-12 13:58:37:858 21328 OCC::SyncJournalDb::commitInternal: void OCC::SyncJournalDb::commitInternal(const QString&, bool) Transaction commit  "update database structure: add inode index" and starting new transaction
    02-12 13:58:37:858 21328 OCC::SyncJournalDb::commitInternal: void OCC::SyncJournalDb::commitInternal(const QString&, bool) Transaction commit  "update database structure: add path index" and starting new transaction
    02-12 13:58:37:858 21328 OCC::SyncJournalDb::tableColumns: Columns in the current journal:  ("path", "lastTryEtag", "lastTryModtime", "retrycount", "errorstring", "lastTryTime", "ignoreDuration", "renameTarget")
    02-12 13:58:37:858 21328 OCC::SyncJournalDb::commitInternal: void OCC::SyncJournalDb::commitInternal(const QString&, bool) Transaction commit  "checkConnect End"

Nothing happens after this even if leaving the client on for hours.

**Comment by hodyroff** [comment]
Ok. Recommend to reproduce with ownCloud Server and then it can be looked at, otherwise we just need to see if it happens again. Reliable reproduction is important to be able to fix things.

**Comment by jaywink** [comment]
@hodyroff I don't believe this is a server issue. The client hangs immediately after adding the local folder. That is why I logged it in the client issue tracker to begin with.

Maybe it is some funky issue with my NFS mount. If someone knows how to debug those I could try provide some information. For now I just copied the files to a local HD instead.
